data/transport missions.txt

```diff
@@ -2176,7 +2176,7 @@ mission "Terraforming 6"
 				`	(Run away.)`
 					flee
 			
-			`	"The unlawful use of an asteroid, Captain. I'm not entirely sure what that means, but I've been told to send you to <planet>. Your compliance would be much appreciated."`
+			`	"The unlawful use of an asteroid, Captain. I'm not entirely sure what that means, but I've been told to send you to <planet> by <day>. Your compliance would be much appreciated."`
```

Comment:
"I've been told to send you to <planet> " is perfect English. "Send you to <planet> by Tuesday" is not, though. I'd suggest making it a separate sentence, e.g. "I've been told to send you to <planet>. They will be expecting you by <day>".

Reply:
This will read (for the correct date): `. . . send you to Earth by the 22nd of January`

Reply:
Either way, I'm not sure that's idiomatic English (but may be wrong, I am not a native, just an English major - but I've never heard "send you to X by the 1st of June")

Reply:
I think this is fine--both the original and the proposed change.